pkg/rancher-components/src/components/Pill/StatusIndicator/types.ts

```diff
@@ -0,0 +1,7 @@
+export type Shape = 'disc' | 'horizontal-bar' | 'vertical-bar';
+export type Status = 'info' | 'success' | 'warning' | 'error' | 'unknown' | 'none';
```

Comment:
nit: Other components like `BadgeState` and `Banner` use a `color` prop to specify similar behavior. 

I quite like the usage of `status`. My request is that we consider consistency moving forward so that we can consistent interfaces between components that specify similar behaviors - so, if `status` is preferred, I'd like to evaluate how we can update other components to utilize `status` as well.